### PR TITLE
os.free nil pointer failure in ngt cgo due to create index hang up

### PR DESCRIPTION
**Pull request by kpango**
Signed-off-by: kpango <kpango@vdaas.org>

### Description:
this PR includes 2 bugfix & 2 feature
bugfix
1. os.free nil pointer failure in ngt cgo due to create index hang up
2. grpc client stream close error

feature
1. show configuration and version information on debug print when server startup
2. rewrite godoc badge.

### Environment:

- Go Version: 1.15.2
- Docker Version: 19.03.8
- Kubernetes Version: 1.18.2
- NGT Version: 1.12.1

### Types of changes:

- [x] Bug fix [type/bug]
- [ ] New feature [type/feature]
- [ ] Add tests [type/test]
- [ ] Security related changes [type/security]
- [ ] Add documents [type/documentation]
- [ ] Refactoring [type/refactoring]
- [ ] Update dependencies [type/dependency]
- [ ] Update benchmarks and performances [type/bench]
- [ ] Update CI [type/ci]

### Changes to Core Features:

- [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully ran tests with your changes locally?

### Checklist:

- [x] I have read the [CONTRIBUTING](https://github.com/vdaas/vald/blob/master/CONTRIBUTING.md) document.
- [x] I have checked open [Pull Requests](https://github.com/vdaas/vald/pulls) for the similar feature or fixes?
- [ ] I have added tests and benchmarks to cover my changes.
- [ ] I have ensured all new and existing tests passed.
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Makefile.d/client.mk [hunk 1]
```diff
@@ -37,9 +37,9 @@ endif
...
 ## insert randomized vectors using valdcli and xpanes
 valdcli/xpanes/insert:
-	xpanes -c "valdcli rand-vecs -n $(NUMBER) -d $(DIMENSION) --with-ids | valdcli -h $(HOST) -p $(PORT) stream-insert --elapsed-time" $$(seq 1 $(NUMPANES))
+	xpanes -c "valdcli rand-vecs -n $(NUMBER) -d $(DIMENSION) --gaussian --gaussian-mean $(MEAN) --gaussian-stddev $(STDDEV) --with-ids | valdcli -h $(HOST) -p $(PORT) stream-insert --elapsed-time" $$(seq 1 $(NUMPANES))
 
 .PHONY: valdcli/xpanes/search
 ## search randomized vectors using valdcli and xpanes
 valdcli/xpanes/search:
-	xpanes -c "valdcli rand-vecs -n $(NUMBER) -d $(DIMENSION) | valdcli -h $(HOST) -p $(PORT) stream-search --elapsed-time" $$(seq 1 $(NUMPANES))
+	xpanes -c "valdcli rand-vecs -n $(NUMBER) -d $(DIMENSION) --gaussian --gaussian-mean $(MEAN) --gaussian-stddev $(STDDEV) --with-ids | valdcli -h $(HOST) -p $(PORT) stream-search --elapsed-time" $$(seq 1 $(NUMPANES))
```
Comment (rinx) [inline Makefile.d/client.mk:45]:
```suggestion
	xpanes -c "valdcli rand-vecs -n $(NUMBER) -d $(DIMENSION) --gaussian --gaussian-mean $(MEAN) --gaussian-stddev $(STDDEV) | valdcli -h $(HOST) -p $(PORT) stream-search --elapsed-time" $$(seq 1 $(NUMPANES))
```

File: pkg/gateway/lb/usecase/vald.go [hunk 2]
```diff
@@ -14,6 +14,7 @@
 // limitations under the License.
 //
 
+// Package usecase reporesents gateways usecase layer
```
Comment (rinx) [inline pkg/gateway/lb/usecase/vald.go:17]:
```suggestion
// Package usecase represents gateways usecase layer
```

**Review by rinx — COMMENTED** [review]
i'll check it on my local cluster.

**Review by rinx — APPROVED** [review]
doesn't work in my local environment but it may be caused by the environmental problem.

**Comment by kpango** [comment]
In addition to the comments, @rinx said, v0.0.66 also has this potential problem which we will have to check more in the environment later.

**Pull request merged**